### ISSUE TITLE
Review english translation

### ISSUE THREAD

**Issue by mcastillof** [mcastillof/FakeTraveler#8]
Hello, I have migrated all strings to strings.xml, so now it will be easier to do translations. I need help to review the [english translation](https://github.com/mcastillof/FakeTraveler/blob/e528b7839015a97d674803365712e557b08df330/app/src/main/res/values/strings.xml) as it is not my first language. Here you have a [spanish translation](https://github.com/mcastillof/FakeTraveler/blob/fea4de78890f2658a69fc3d6fe5150238008857c/app/src/main/res/values-es/strings.xml), maybe could be of help.

**Comment by hanelore** [comment]
Hello @mcastillof English translation in reviewed.

Thanks again :)

**Comment by rosa2** [comment]
Hello
Any instructions on how to translate? Thanks

**Comment by ThexXTURBOXx** [comment]
Hello, you just need to translate this file and create a PR containing the new strings (in a file such as `app/src/main/res/values-NEWLANGUAGECODE/strings.xml`):
https://github.com/mcastillof/FakeTraveler/blob/master/app/src/main/res/values/strings.xml

**Comment by ThexXTURBOXx** [comment]
Yes, looks good! :)

**Comment by rosa2** [comment]
> Hello, you just need to translate this file and create a PR containing the new strings (in a file such as `app/src/main/res/values-NEWLANGUAGECODE/strings.xml`): https://github.com/mcastillof/FakeTraveler/blob/master/app/src/main/res/values/strings.xml

Thanks.
I am still organizing apps that need translation in F-Droid.
F-Droid has a link to the translation in each app.
Your link is not there
https://f-droid.org/en/packages/cl.coders.faketraveler/
as it is in others, like Organic Maps (just before Source Code)
https://f-droid.org/en/packages/app.organicmaps/
I have just reported, but I don't have the knowledge to solve it.
https://github.com/agateau/pixelwheels/issues/481#issuecomment-2693903966
Maybe is something in metadata?
Could you please check?
Thanks

**Comment by ThexXTURBOXx** [comment]
Yes, in the metadata, one can specify a `Translation` tag. However, I would advise only specifying that string when you have a dedicated translation system such as POEditor or Crowdin (which FakeTraveler does not have).
So, it does not make much sense to specify it there

**Comment by rosa2** [comment]
As a translator, any link would be good for me. Also, to see that link means to us that you can translate the app.
If you could make it, I think that the app would get more translations.
I teach the app in workshops. This week to students around 14 years. To them I teach that you can collaborate with FLOSS to translate, so maybe one would, if I don't have the time before.
And at last, could you please point to me in which file you write that metadata? Like that, if I find other app with this problem, I can search if it is the reason. 
Thanks again for all :)

**Comment by rosa2** [comment]
The follow up in F-Droid now is in a [different issue.](https://gitlab.com/fdroid/fdroiddata/-/issues/3490#note_2375502705) 2 apps where solved and yours is waiting.

**Comment by ThexXTURBOXx** [comment]
> As a translator, any link would be good for me. Also, to see that link means to us that you can translate the app.
> If you could make it, I think that the app would get more translations.

On F-Droid, pretty much 99% of the apps can be translated, since their source code is public anyway.
Whether or not PRs will be accepted for translations is up to the maintainer (but that is also the case for general contributions).
The problem is that you could not only translate the app itself, but also the entry on F-Droid, which is located at an entirely different location: https://github.com/mcastillof/FakeTraveler/tree/master/fastlane/metadata/android
Hence, I would stay away from specifying just a single link for translation.

Another problem is: where would that link point to? Just https://github.com/mcastillof/FakeTraveler/blob/master/app/src/main/res/values/strings.xml? But this would be a very incomplete link as it only points to a file, but not a translation interface.
Better would be something like https://github.com/mcastillof/FakeTraveler/edit/master/app/src/main/res/values/strings.xml
However, that just edits the English translation...
As already said, without a dedicated translation interface, a "Translation" link on F-Droid is rather useless, I think.

> And at last, could you please point to me in which file you write that metadata? Like that, if I find other app with this problem, I can search if it is the reason.

You would need to specify it in this file: https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/cl.coders.faketraveler.yml by adding a `Translation: [URL_here]` entry
